src/ArrayValueCalculator.java

```diff
@@ -0,0 +1,33 @@
+public class ArrayValueCalculator {
+
+    public static String doCalc(String stringArray[][]) throws ArraySizeException, ArrayDataException {
+        int sum = 0;
+
+        for (int i = 0; i < stringArray.length; i++) {
+            for (int j = 0; j < stringArray[i].length; j++) {
+                if (i <= 0 || j <= 0) {
+                    throw new IllegalArgumentException("Each element in the array must be non-negative and greater than zero");
```

Comment:
Sorry, but this is wrong. Any element can be lower or equal to 0. Just think of it: can you do any operations via negative numbers or 0s on a common calculator?